pandas/tests/arrays/masked/test_arithmetic.py

```diff
@@ -162,3 +162,16 @@ def test_error_len_mismatch(data, all_arithmetic_operators):
         s = pd.Series(data)
         with pytest.raises(ValueError, match="Lengths must match"):
             op(s, other)
+
+
+@pytest.mark.parametrize("op", ["__neg__", "__abs__", "__invert__"])
+@pytest.mark.parametrize(
+    "values, dtype", [([1, 2, 3], "Int64"), ([True, False, True], "boolean")]
```

Comment:
you can use the `data` fixture here as used in the other tests above

Reply:
this is being addressed in #39916, so no action required here